src/Producer/Producer.php

```diff
@@ -53,7 +53,7 @@ public function __construct(
 	}
 
 
-	public function publish(string $message, array $headers = []): void
+	public function publish(string $message, array $headers = [], ?string $routingKey = null): void
```

Comment:
Shouldn't you define the routing key on the producer-config-level?

Reply:
Definitely not, routing keys are properties of individual messages

Reply:
ok